scripts/centos-req.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+sudo yum groupinstall -y "Development tools"
+sudo yum install -y gmp-devel mpfr-devel libmpc-devel zlib-devel vim git java java-devel
+curl https://bintray.com/sbt/rpm/rpm | sudo tee /etc/yum.repos.d/bintray-sbt-rpm.repo
+sudo yum install -y sbt texinfo gengetopt
+sudo yum install -y expat-devel libusb1-devel ncurses-devel cmake "perl(ExtUtils::MakeMaker)"
+# deps for poky
+sudo yum install -y python36 patch diffstat texi2html texinfo subversion chrpath git wget
+# deps for qemu
+sudo yum install -y gtk3-devel
+# deps for firemarshal
+sudo yum install -y python36-pip python36-devel rsync
+# Install GNU make 4.x (needed to cross-compile glibc 2.28+)
+sudo yum install -y centos-release-scl
+sudo yum install -y devtoolset-8-make
+
+# install DTC
+DTCversion=dtc-1.4.4
+wget https://git.kernel.org/pub/scm/utils/dtc/dtc.git/snapshot/$DTCversion.tar.gz
+tar -xvf $DTCversion.tar.gz
+cd $DTCversion
+make -j16
+make install
```

Comment:
CentOS Extras provides `dtc` (1.4.6 with CentOS 7).
Otherwise, sudo is needed to install to the `/usr/local` prefix.
```suggestion
sudo make install
```